plugins/modules/docker_config.py

```diff
@@ -92,6 +98,7 @@
 author:
   - Chris Houseknecht (@chouseknecht)
   - John Hu (@ushuz)
+  - Sasha Jenner (@sashajenner)
```

Comment:
Please note that by adding yourself here, you sign up as a maintainer of this module.

Reply:
I would rather not do that, thanks for letting me know.